.circleci/config.yml

```diff
@@ -254,6 +254,7 @@ jobs:
               . activate habitat;
               cd habitat-sim
               pip install -r requirements.txt --progress-bar off
+              pip install imageio imageio-ffmpeg
```

Comment:
This should be added to `requirements.txt` instead of modifying the CI.  Users need to install imageio to import the package without error.  This will also need to be added to the conda-build setup.